### ISSUE TITLE
Revise How to add markup tutorial

### ISSUE THREAD

**Issue by AlasdairGray** [BioSchemas/specifications#514]
At the [community call on 28 June 2021](https://docs.google.com/document/d/1kd5F97ogdiPNhLTnkei-RVR8TC8Ohpc5QSPX3KsfDrk/edit#heading=h.98egkfgjx50f) the following issues were identified with a brief scan of the [How to add markup tutorial](https://bioschemas.org/tutorials/howto/howto_add_markup):
- It's not long enough (@nsjuty / @AlasdairGray )
- Missing statement on **where** the content needs to be added, and **what** creates that HTML (@egonw )
- Where to put the JSON-LD in the HTML page? Multiple places in cases of multiple objects (@egonw )

_Originally posted by @AlasdairGray in https://github.com/BioSchemas/specifications/issues/488#issuecomment-870733495_

**Comment by AlasdairGray** [comment]
- [ ] @AlasdairGray to add content from markupathon slides
- [ ] @ivanmicetic to add experience from IDP use case 
- [ ]  @alaninmcr to add experience from ComputationalWorkflow perspective
- [ ] @albangaignard to add experience from ComputationalTool perspective

The different perspectives may be extracted out into dedicated recipes for those communities.

**Comment by AlasdairGray** [comment]
There is probably cross-over with the content of the [How to select the right profile for your resource](https://bioschemas.org/tutorials/howto/howto_right_profile)

**Comment by ljgarcia** [comment]
I think the different perspectives are more for [How to select the right profile for your resource](https://bioschemas.org/tutorials/howto/howto_right_profile) than for [How to add markup](https://bioschemas.org/tutorials/howto/howto_add_markup).

Related issues:
* ComputationalTool https://github.com/BioSchemas/specifications/issues/547
* Chemical profiles https://github.com/BioSchemas/specifications/issues/554

**Comment by ljgarcia** [comment]
The How to add markup tutorial has been updated and is ready for review, see https://github.com/BioSchemas/bioschemas.github.io/pull/447

**Comment by AlasdairGray** [comment]
> I think the different perspectives are more for [How to select the right profile for your resource](https://bioschemas.org/tutorials/howto/howto_right_profile) than for [How to add markup](https://bioschemas.org/tutorials/howto/howto_add_markup).
> 
> Related issues:
> 
> * ComputationalTool [Add paragraph about ComputationTool to Choosing the Right Profile tutorial #547](https://github.com/BioSchemas/specifications/issues/547)
> * Chemical profiles [Add paragraph about chemical profiles to Choosing the Right Profile tutorial #554](https://github.com/BioSchemas/specifications/issues/554)

I agree, can we move these into a separate issue for that tutorial. Once that is done this issue can be closed.
